### ISSUE TITLE
Video about CV?

### ISSUE THREAD

**Issue by mxmilkiib** [sjaehn/lv2tutorial#8]
Might you be up for including the implementation of [CV](https://linuxmusicians.com/viewtopic.php?f=1&t=20701) in a future video?

**Comment by sjaehn** [comment]
I was already waiting for this question from you ;-). Sure, it will come. But now sure when and how to demonstrate. Maybe a CV LFO. But there are still some extensions with higher priority (state, worker). And don't forget the LV2UI!

**Comment by mxmilkiib** [comment]
An LFO would be grand, output would be easiest, input to control it harder, and harder still I guess would be to affect, say, the ADSR of the synth. Just some 2p thoughts, do what you think is best :)

**Comment by sjaehn** [comment]
Yes output will be easy. Something to start with. But also the others aren't that complicated. Will be in the third block of tutorial videos. But first, I'll cover the UI topic in a second block in the new year. BTW, I should do a road map about the tutorial video project.
